lib/jobs/ansible-job.js

```diff
@@ -65,6 +86,23 @@ function ansibleJobFactory(
             return self._publishAnsibleResult(self.routingKey, data);
         })
         .then(function() {
+            try {
+                var data = fs.readFileSync(self.outFileName, 'utf8');
```

Comment:
@stevemendes  based on your previous comments with @yyscamper , so fs.readFileAsync has some problems?  a example for it is https://github.com/RackHD/on-http/blob/master/lib/services/schema-api-service.js#L41